Composer/packages/client/src/pages/design/exportSkillModal/constants.tsx

```diff
@@ -21,10 +21,10 @@ import {
   SelectTriggers,
 } from './content';
 
-export const VERSION_REGEX = /\d\.\d\.(\d+|preview-\d+)/i;
+export const VERSION_REGEX = /\d\.\d(\.\d+|preview-\d+)|\d\.\d/i;
```

Comment:
This needs a tweak to handle version strings like "1.10" (put a `+` after the second `d` and that should work).

Reply:
Good call